doc/overview.md

```diff
@@ -0,0 +1,101 @@
+
+The Translator sends the SAML AuthnResponse to the Verify Service Provider (VSP) to perform the SAML validation. The Translator service will generate an eIDAS AuthnResponse based on the attributes it receives from the VSP. The Translator service then encrypts the signed assertion for the connector node and sends the whole eIDAS SAML response for signing by the Signing service.
+
+Once the eIDAS SAML response is signed, the Translator service passes it on to the Gateway service.
+
+### Verify-Service-Provider
+
+The Verify Service Provider (VSP) generates and translates Security Assertion Markup Language (SAML) messages to and from the GOV.UK Verify Hub. 
+The Gateway service uses the VSP to generate SAML requests to send to the Verify Hub. For responses, the Translator service uses the VSP to validate SAML responses and translate them to JSON. 
+
+### Signing-Service
+
+This component is a microservice running on an Amazon EC2 instance. The service will use an Amazon hardware security module (HSM) to sign outbound eIDAS SAML messages. 
+
+## Connecting to Countries
+
+We will host one instance of the proxy node services and infrastructure isolation per requesting Country connector node. All instances will share a single hardware security module (HSM) but will each have independent credentials to access the HSM, and an independent signing key held by the HSM.
+
+## Infrastructure
+
+The eIDAS Proxy Node is hosted on an Amazon AWS cloud infrastructure and runs on Kubernetes. This is isolated from all Verify Hub deployments to ensure that it is treated as a separate component. Public and private subnets are used to isolate the Translator and Signing services. Access to environments and deployments is controlled using AWS’s identity and access management permissions.
+
+## Security operations monitoring
+
+Security operations within the organisation will monitor transactions and access. Transaction monitoring will involve correlating messages entering with messages exiting the proxy node. Access monitoring will be put in place using Amazon's CloudWatch logging. Logs will be monitored by a Security Operations team to track access to servers, deployments and key rotations.
+
+## UK Proxy Node key management
+
+SAML responses going to EU member state connector nodes will be signed with keys stored on an HSM. These are keys with self-signed certificates. The same key will be used to sign the proxy node metadata. 
+
+The Verify Service Provider will be responsible for Signing AuthnRequests and Encrypting Responses from the Verify Hub. These will be stored as sealed secrets. 
+
+| 			Key type           | 		Key location        |
+| ---------------------------- | -------------------------- |
+| Metadata signing key         | AWS Key Management System  |
+| AuthnRespone signing key     | AWS Key Management System  |
+| AuthnRequest signing key     | Verify Service Provider    |
```

Comment:
I'm not sure from this how the EU member state will validate the signature created by the CloudHSM signing keys.
Do we publish the CloudHSM signing certs. I might have missed something from above ☝️ though

Reply:
We publish the signing cert in the Proxy Node Metadata, which they will consume.